test/testleakautovar.cpp

```diff
@@ -631,6 +634,26 @@ class TestLeakAutoVar : public TestFixture {
         ASSERT_EQUALS("", errout_str());
     }
 
+    void memcpyWithPtr1() { // #11542
+        check("#include <string.h>\n"
+              "void f(char** old, char* value) {\n"
+              "    char *str = strdup(value);\n"
+              "    memcpy(old, &str, sizeof(char*));\n"
+              "}\n");
+        ASSERT_EQUALS("", errout_str());
+    }
+
+    void memcpyWithPtr2() {
+        check("#include <string.h>\n"
+              "void f(char* value) {\n"
+              "    char *old = NULL;\n"
+              "    char *str = strdup(value);\n"
+              "    memcpy(&old, &str, sizeof(char*));\n"
+              "}\n");
+        // TODO: make this fail
+        ASSERT_EQUALS("", errout_str());
```

Comment:
Running cppcheck on this code on the command line (without any additional options) produces an error,but nothing happens in the test function.

What could the reason for this be?

Reply:
Maybe `std.cfg` is not loaded in that test, so `strdup()` and `memcpy()` are not recognized. There should be other tests  where we load the library.

Reply:
I'm just matching the token with `"memcpy"` though, is the library loading really required for that?